schema/Electrometer.yaml

```diff
@@ -8,12 +8,12 @@ properties:
     description: The path of where the copied fits file are located.
     type: string
     default: "~/electrometerFitsFiles"
-  ip_address:
+  server_address:
```

Comment:
Please consider `file_server_address` and `file_server_port` for clarity.

Without the prefix folks may think it is the address of the Electrometer low-level controller. (Which it may well be someday -- if we put the serial port on a terminal server).